### ISSUE TITLE
`gilbert render` traceback

### ISSUE THREAD

**Issue by shuttle1987** [funkybob/gilbert#35]
Sometime in recent commits I have found this to be happening:
```
$ gilbert render
Traceback (most recent call last):
  File "/home/janis/digital_site/venv/bin/gilbert", line 11, in <module>
    load_entry_point('gilbert==0.4.1', 'console_scripts', 'gilbert')()
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2843, in load_entry_point
    return ep.load()
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2434, in load
    return self.resolve()
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2440, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/gilbert/__init__.py", line 1, in <module>
    from .site import Site  # NOQA
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/gilbert/site.py", line 10, in <module>
    from .collection import Collection
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/gilbert/collection.py", line 3, in <module>
    from .content import Content
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/gilbert/content.py", line 60, in <module>
    class Raw(Content):
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/gilbert/schema.py", line 153, in __new__
    return type.__new__(mcs, classname, bases, namespace, **kwargs)
  File "/home/janis/digital_site/venv/lib/python3.7/site-packages/gilbert/content.py", line 35, in __init_subclass__
    cls._types[cls.__name__] = cls
TypeError: 'SchemaProperty' object does not support item assignment

```

**Comment by shuttle1987** [comment]
For some more info, commit 00f3a91e38bcc9ab9eadd5b412c35ba112f6e6c5 will render the exact same files as in the original issue post without a problem.

**Comment by funkybob** [comment]
Is this error against gilbert 0.4.1 ?  or master? or another version?

**Comment by shuttle1987** [comment]
This error occurred installing against master, the specific commit from gilbert was  9240d5a78424e2b94ea2f54e1a8da44b027a5440

**Comment by funkybob** [comment]
I believe I've corrected the problem in 1febb05

**Comment by funkybob** [comment]
Closing - assumed fixed?
